### ISSUE TITLE
Typo in Invoke-SFShutdown help

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#49]
```
PS /home/sean> get-help Invoke-SFShutdown -examples                             

NAME
    Invoke-SFShutdown
    
SYNOPSIS
    [Cluster] Shutdown one or more nodes in a cluster.
    
    
    -------------------------- EXAMPLE 1 --------------------------
    
    PS C:\>Invoke-SFShutdown -NodeID 1,2,3,4
    
    Shuts down nodes 1 thorugh 4 <================ 

```

through is misspelled.

**Comment by Salmanius** [comment]
Got this alongside issue #52 , both will fixed in 1.6.
Thank you.

**Comment by arjun960** [comment]
@scaleoutsean We have addressed and resolved this issue and the fix will be available in the next SDK release.
